typescript-light-bulb/main.ts

```diff
@@ -0,0 +1,21 @@
+const $lightBulb = document.querySelector('.light-bulb');
+const $container = document.querySelector('.container');
+
+if (!$lightBulb) throw new Error('$lightBulb query failed');
+if (!$container) throw new Error('$container query failed');
+
+$lightBulb.addEventListener('click', () => {
+  if ($lightBulb.className === 'light-bulb off') {
+    $lightBulb.className = 'light-bulb on';
+  } else {
+    $lightBulb.className = 'light-bulb off';
+  }
+});
+
+$container.addEventListener('click', () => {
+  if ($container.className === 'container off') {
+    $container.className = 'container on';
+  } else {
+    $container.className = 'container off';
+  }
+});
```

Comment:
I would have combined these into just 1 event listener. No need to break them up
```suggestion
$lightBulb.addEventListener('click', () => {
  if ($lightBulb.className === 'light-bulb off') {
    $lightBulb.className = 'light-bulb on';
     $container.className = 'container on';
  } else {
    $lightBulb.className = 'light-bulb off';
    $container.className = 'container off';
  }
});
```

Reply:
ohhh okay! thank you